js/functions.js

```diff
@@ -0,0 +1,41 @@
+
+const calculation = function () {                   // actual calculation when "=" is pressed
+    switch (mathOperation) {
+        case "addition":
+            result += secondNumber;
+            break;
+        case "subtraction":
+            result -= secondNumber;
+            break;
+        case "multiplication":
+            result *= secondNumber;
+            break;
+        case "division":
+            result /= secondNumber;
+            break;
+        case "divisionWithoutRemainder":
+            result %= secondNumber;
+            break;
+        default:
+            screen.value = "ERROR";   
+        }
+};
+
+const eraseZeroInputtedAtBeginningOfInteger = function () {
+    if (screen.value.charAt(0) === "0" && screen.value.charAt(1) !== "." ) {
+        newNumber = true;
+    }
+}
+
+const inputDigitDecidingIfItIsNewNumber = function (symbol) {
+    if (dropResult) {
+        result = null;
+        dropResult = false;
+    }
+    screen.value = newNumber ? symbol : screen.value + symbol;
+    newNumber = false;
```

Comment:
variables that store boolean type better name with prefix `is` or `has`, example: `isNewNumber`